sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkSessionClient.cs

```diff
@@ -28,5 +31,285 @@ public SparkSessionClient(Uri endpoint, string sparkPoolName, TokenCredential cr
                   options.VersionString)
         {
         }
+
+        /// <summary> Gets a single spark session. </summary>
+        /// <param name="sessionId"> Identifier for the session. </param>
+        /// <param name="detailed"> Optional query param specifying whether detailed response is returned beyond plain livy. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        internal virtual async Task<Response<SparkSession>> GetSparkSessionAsync(int sessionId, bool? detailed = null, CancellationToken cancellationToken = default)
```

Comment:
Why is this internal?

Reply:
Called by UpdateStatusAsync on SparkSessionOperation.